src/Resources/config/services.yml

```diff
@@ -12,12 +12,12 @@ services:
 
     ThreeBRS\SortingPlugin\Controller\SortingController:
         public: true
-        arguments:
-            $templatingEngine: '@twig'
-            $taxonRepository: '@sylius.repository.taxon'
-            $productTaxonRepository: '@sylius.repository.product_taxon'
-            $entityManager: '@doctrine.orm.entity_manager'
-            $eventDispatcher: '@event_dispatcher'
-            $router: '@router'
-            $translator: '@translator'
-            $flashBag: '@session.flash_bag'
+        autowire: true
+
+    ThreeBRS\SortingPlugin\Service\ProductPositionsUpdater:
+        public: true
+        autowire: true
```

Comment:
nechal bych false
je lepší když ve vendorech je vidět, to se do konstruktoru (servisky) dostává

Reply:
Jestli se nepletu, tak autowire true znamená, že tuhle service nemusí závislý kód výslovně uvádět jako závislost v konfiguraci - prostě uvede závislost v konstruktoru.

Máš na mysli, že ti přijde lepší, když jsou všechny závislosti výslovně vypsané v configách?